dev-util/capstone/capstone-4.0.1.recipe

```diff
@@ -65,3 +71,10 @@ INSTALL()
 	fixPkgconfig
 	packageEntries devel $developDir
 }
+
+TEST()
+{
+	wget ${fuzz_test_url}/raw/master/corpus/${fuzz_test_file}
```

Comment:
Please don't download within the recipe. This will cause it to not be cached and would be redownloaded everytime `TEST()` is run.

If you want to download the code at a specific commit, use https://github.com/groundx/capstonefuzz/archive/<commit hash>.tar.gz

Reply:
Without the zipfile the test fails with make

Reply:
Using srcGitRev="commit" and SOURCE_URI_2 will fix this, no need for wget

Reply:
SOURCE_URI_2 would mean this file is needed to build the project. it is not.

Reply:
Got me on that one, then we need another way to download this just when the test is invoked

Reply:
As discussed on IRC, we have decided to go with downloading the fuzz tests file using SOURCE_URI_2 which is downloaded once on every clean build rather than every-time TEST() is run.